File: ToughDashApr19/four.cpp
#include<iostream>

using namespace std;

int main() {
    string input;

    unsigned short int t, i, j, len, ball, over;

    cin >> t;

    for(i = 0; i < t; i++)
    {
        cin >> input;
        len = input.length();
        ball = 0;

        for(j = 0; j < len; j++)
        {
            if ((input.at(j) >= '0' && input.at(j) <= '6') || input.at(j) == 'O') //validity of a ball
                ball++;            
        }
        over = ball/6;
        ball = ball % 6;
        
        if(over > 1)
            cout << over << " OVERS ";
        else if(over == 1)
            cout << over << " OVER ";
        
        if(ball > 1)
            cout << ball << " BALLS";
        else if(ball == 1)
            cout << ball << " BALL";
        cout << endl;
        
    }
    

    return 0;
}

File: ToughDashApr19/four.py
t = int(input())

for x in range(t):
    line = input()
    balls = 0
    length = len(line)
    
    # items = [line[y] for y in range(length)]
    for i in range(length):
        if '0' <= line[i] <= '6' or line[i] == 'O':
            balls += 1

    over = balls // 6
    balls = balls % 6

    if over > 1:
        print(over,"OVERS ", end ='') # prints the line without \n, new line
    elif over == 1:
        print("1 OVER ", end = '')
    
    if balls > 1:
        print(balls,"BALLS", end = '')
    elif balls == 1:
        print("1 BALL", end = '')
    
    print()

